src/plugins/saml-auth/index.js

```diff
@@ -0,0 +1,87 @@
+import SamlAuth from "./components/SamlAuth"
+import WrappedSamlAuthItem from "./components/WrapAuthItem"
+import AuthorizationPopup from "./components/AuthorizationPopup"
+import * as actions from "./actions"
+import * as selectors from "./selectors"
+import reducers from "./reducers"
+
+const getAuthToken = (search, authTokenSearchName = "token") => {
+  return search
+    .substring(1)
+    .split("&")
+    .reduce((queries, keyValue) => {
+      const [key, value] = keyValue.split("=")
+      return { ...queries, [key]: value }
+    }, {})[authTokenSearchName]
+}
+
+function getSchema(system, key) {
+  const {
+    specSelectors: { specJson },
+  } = system
+
+  const definitionBase = ["securityDefinitions"]
+  const schema = specJson().getIn([...definitionBase, key])
+
+  return schema
+}
+
+let engaged = false
+const samlAuthPlugin = () => {
```

Comment:
would you happen to have the link to swaggerui documentation explaining what this plugin consists of?